packages/components/psammead-sitewide-links/src/index.jsx

```diff
@@ -36,6 +37,9 @@ const SitewideLinksWrapper = styled.div`
 const ConstrainedWrapper = styled.div`
   max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
   margin: 0 auto;
+  ${({ isTrustProject }) => {
+    return isTrustProject !== 'object' && `padding-top: ${GEL_SPACING}`;
```

Comment:
Why do we have to check if TrustProjectLink is an object? should this not be caught by the proptypes?